src/index.js

```diff
@@ -1,15 +1,17 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
 import reportWebVitals from './reportWebVitals';
```

Comment:
- [ ] This are boilerplate you don't  need them in your application
please delete them